build/Helix/NuGet.config

```diff
@@ -19,7 +19,8 @@
     <clear />
     <add key="NuGet official package source" value="https://nuget.org/api/v3/" />
     <add key="dotnetfeed" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
-    <add key="WindowsES-External" value="https://microsoft.pkgs.visualstudio.com/_packaging/WindowsES-External/nuget/v3/index.json" />	
+    <add key="WindowsES-External" value="https://microsoft.pkgs.visualstudio.com/_packaging/WindowsES-External/nuget/v3/index.json" />
```

Comment:
do we still need this? also we can remove those authenticate steps

Reply:
Yea we don't